app/views/subtopics/show.html.erb

```diff
@@ -1,20 +1,14 @@
-<% content_for :title, subtopic.combined_title %>
+<% content_for :title, "#{subtopic.title}: detailed information" %>
 <%= render 'shared/tag_meta', tag: subtopic %>
 <% content_for :meta_tags do %>
   <meta name='govuk:navigation-page-type' content='Second Level Topic'>
   <meta name="govuk:section" content="<%= meta_section %>">
 <% end %>
 <% content_for :page_title do %>
-  <%
-    title_params = { title: subtopic.title }
-    if subtopic.parent
-      title_params[:context] = {
-        text: subtopic.parent.title,
-        href: subtopic.parent.base_path
-      }
-    end
-  %>
-  <%= render "govuk_publishing_components/components/title", title_params %>
+  <%= render "govuk_publishing_components/components/title", { 
+    title: subtopic.title,
+    context: "detailed information",
```

Comment:
Should `detailed` be capitalised?

```suggestion
    context: "Detailed information",
```

Reply:
This is something I wasn't 100% on. The card uses lowercase but I'll check with a content person.